testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -890,6 +889,15 @@ def test_persistent_offsets_readonly(self, tmpdir):
         assert_equal(os.path.exists(XDR.offsets_filename(filename,
                                                     ending='.lock')), False)
 
+        # pre-teardown permission fix - leaving permission blocked dir
```

Comment:
add reference to issue number, please --- so that we can maybe remove again when it works again